src/modules/appointment/appointment.service.ts

```diff
@@ -205,6 +207,63 @@ export default class AppointmentService {
     }
   }
 
+  async startAppointments() {
```

Comment:
what is with types? have you checked PR with checklist? I spend months to gather al rules together, and you just ignore them